net/lp_rpc.proto

```diff
@@ -154,3 +157,56 @@ message NotifySegment {
     int64 taskId   = 16;
     bytes profiles = 17;
 }
+
+// Required parameters for probabilistic micropayment tickets
+message TicketParams {
+    // Pay out to the recipient if the ticket wins
+    bytes face_value = 1;
+
+    // Probability that the ticket wins
+    bytes win_prob = 2;
+
+    // 32 byte keccak-256 hash commitment to a random number provided
+    // by the recipient
+    bytes recipient_rand_hash = 3;
+
+    // Value generated by recipient that the recipient can use
+    // to derive the random number corresponding to the recipient's hash commitment
+    bytes seed = 4;
+}
+
+// Probabilistic micropayment ticket
+message Ticket {
+    // ETH address of the recipient
+    string recipient = 1;
+
+    // ETH address of the sender
+    string sender = 2;
```

Comment:
Curious, why `string` and not `bytes` for ETH addresses?  Note that we already use `bytes` within the `OrchestratorRequest` struct itself.

https://github.com/livepeer/go-livepeer/blob/59dd08af477d95bce7869a4d8636759038f86e97/net/lp_rpc.proto#L29-L32

Reply:
For some reason I thought we were using `string` for ETH addresses - must've misread. I'll make the change.